raiden/network/transport/udp/udp_transport.py

```diff
@@ -337,8 +245,7 @@ def init_queue_for(
         events = self.get_health_events(recipient)
 
         greenlet_queue = gevent.spawn(
-            single_queue_send,
-            self,
+            self.single_queue_send,
```

Comment:
same thing with `single_queue_send`, the only necessary argument is the target address